templates/typescript/samples/sample-skill/src/cognitivemodels.json

```
@@ -13,7 +13,7 @@
           "subscriptionKey": ""
         },
         {
-          "id": "skill",
+          "id": "skillSample",
```

```suggestion
          "id": "sampleSkill",
```

As the solution is `sample-skill`